apis/project.cattle.io/v3/pipeline_types.go

```diff
@@ -272,17 +272,17 @@ type ApplyYamlConfig struct {
 }
 
 type PublishCatalogConfig struct {
-	Path      string `json:"path,omitempty" yaml:"path,omitempty"`
-	Catalog   string `json:"catalog,omitempty" yaml:"catalog,omitempty"`
-	Version   string `json:"version,omitempty" yaml:"version,omitempty"`
-	GitURL    string `json:"gitUrl,omitempty" yaml:"gitUrl,omitempty"`
-	GitBranch string `json:"gitBranch,omitempty" yaml:"gitBranch,omitempty"`
-	GitAuthor string `json:"gitAuthor,omitempty" yaml:"gitAuthor,omitempty"`
-	GitEmail  string `json:"gitEmail,omitempty" yaml:"gitEmail,omitempty"`
+	Path            string `json:"path,omitempty" yaml:"path,omitempty"`
+	CatalogTemplate string `json:"catalogTemplate,omitempty" yaml:"catalogTemplate,omitempty"`
```

Comment:
Does it mean that we are breaking api compatibility?

Reply:
This is a new feature in 2.2

Reply:
Ok, all good then